### PR TITLE
Fix width of the button to set the variation prices

### LINKED ISSUE #38054: Button for sending the variant prices has a fixed size, but translation could be too long
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

After creating a variant product with an attribute (like size), there is an option to add the same price to every variant which opens a modal. The send button on this modal has a fixed size (in pixel), but in German (and maybe more locales) the string is too long.

Without text:
Bildschirmfoto 2023-04-25 um 11 47 46

With text:
Bildschirmfoto 2023-04-25 um 11 47 54

### Expected behavior

The button width should be set to "auto" in this case.

Screenshot with button css set to `width: auto;`:

Bildschirmfoto 2023-04-25 um 11 49 36

Bildschirmfoto 2023-04-25 um 11 49 24

### Actual behavior

Buttons has fixed pixel size.

### Steps to reproduce

Creating a variant product
Add an attribute (like size) and at least two options
After creating the variants with all attributes there is a new info and button to add the price - click it
The modal is showing with the bug.

Additionally, the button for opening the modal could get some margin, because the text is nearly under it:

Bildschirmfoto 2023-04-25 um 11 53 43

### WordPress Environment

`
### WordPress Environment ###

WordPress address (URL): https://shop.example.de
Site address (URL): https://shop.example.de
WC Version: 7.6.1
REST API Version: ✔ 7.6.1
WC Blocks Version: ✔ 9.8.5
Action Scheduler Version: ✔ 3.5.4
Log Directory Writable: ✔
WP Version: 6.2
WP Multisite: –
WP Memory Limit: ❌ 40 MB - Wir empfehlen das PHP Speicher-Limit (memory_limit) auf mindestens 64 MB zu setzen. Siehe: PHP Speicher-Limit erhöhen
WP Debug Mode: –
WP Cron: ✔
Language: de_DE
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 7.4.33
PHP Post Max Size: 64 MB
PHP Time Limit: 50000
PHP Max Input Vars: 5000
cURL Version: 7.64.0
OpenSSL/1.1.1n

SUHOSIN Installed: –
MySQL Version: 5.7.38-log
Max Upload Size: 64 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 7.6.1
WC Database Prefix: wp_shop_
Datenbank-Gesamtgröße: 6.45MB
Datenbank-Datengröße: 4.09MB
Datenbank-Indexgröße: 2.36MB
wp_shop_woocommerce_sessions: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shop_woocommerce_api_keys: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_woocommerce_attribute_taxonomies: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shop_woocommerce_downloadable_product_permissions: Daten: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_shop_woocommerce_order_items: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shop_woocommerce_order_itemmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_woocommerce_tax_rates: Daten: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_shop_woocommerce_tax_rate_locations: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_woocommerce_shipping_zones: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_shop_woocommerce_shipping_zone_locations: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_woocommerce_shipping_zone_methods: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_shop_woocommerce_payment_tokens: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shop_woocommerce_payment_tokenmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_woocommerce_log: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shop_actionscheduler_actions: Daten: 0.14MB + Index: 0.14MB + Engine InnoDB
wp_shop_actionscheduler_claims: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shop_actionscheduler_groups: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shop_actionscheduler_logs: Daten: 0.11MB + Index: 0.09MB + Engine InnoDB
wp_shop_commentmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_comments: Daten: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_shop_links: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shop_options: Daten: 2.08MB + Index: 0.16MB + Engine InnoDB
wp_shop_postmeta: Daten: 0.50MB + Index: 0.36MB + Engine InnoDB
wp_shop_posts: Daten: 0.13MB + Index: 0.06MB + Engine InnoDB
wp_shop_termmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_terms: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_term_relationships: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shop_term_taxonomy: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_usermeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_users: Daten: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_shop_wc_admin_notes: Daten: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_shop_wc_admin_note_actions: Daten: 0.05MB + Index: 0.02MB + Engine InnoDB
wp_shop_wc_category_lookup: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_shop_wc_customer_lookup: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_wc_download_log: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_wc_order_coupon_lookup: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_wc_order_product_lookup: Daten: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_shop_wc_order_stats: Daten: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_shop_wc_order_tax_lookup: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_wc_product_attributes_lookup: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shop_wc_product_download_directories: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shop_wc_product_meta_lookup: Daten: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_shop_wc_rate_limits: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shop_wc_reserved_stock: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_shop_wc_tax_rate_classes: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shop_wc_webhooks: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shop_woocommerce_gzd_dhl_im_products: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_woocommerce_gzd_dhl_im_product_services: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_woocommerce_gzd_packaging: Daten: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_shop_woocommerce_gzd_packagingmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_woocommerce_gzd_shipmentmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_woocommerce_gzd_shipments: Daten: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_shop_woocommerce_gzd_shipment_itemmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_woocommerce_gzd_shipment_items: Daten: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_shop_woocommerce_gzd_shipment_labelmeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_woocommerce_gzd_shipment_labels: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_shop_woocommerce_gzd_shipping_provider: Daten: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_shop_woocommerce_gzd_shipping_providermeta: Daten: 0.02MB + Index: 0.03MB + Engine InnoDB

### Post Type Counts ###

attachment: 62
custom_css: 1
et_body_layout: 1
et_template: 3
et_theme_builder: 1
nav_menu_item: 7
page: 13
post: 2
product: 28
product_variation: 71
revision: 14
wp_global_styles: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (3) ###

Under Construction: von WebFactory Ltd – 3.97
Germanized für WooCommerce: von vendidero – 3.12.2
WooCommerce: von Automattic – 7.6.1

### Inactive Plugins (2) ###

Germanized for WooCommerce Pro: von vendidero – 3.2.5
Stock Locations for WooCommerce: von Fahad Mahmood & Alexandre Faustino – 2.4.3

### Settings ###

API Enabled: –
Force SSL: –
Currency: EUR (€)
Currency Position: right_space
Thousand Separator: .
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –
Enforce Approved Product Download Directories: ✔
Order datastore: WC_Order_Data_Store_CPT

### WC Pages ###

Shop-Basis: #9 - /shop/
Warenkorb: #10 - /warenkorb/
Kasse: #11 - /kasse/
Mein Konto: #12 - /mein-konto/
Allgemeine Geschäftsbedingungen: #180 - /agb/

### Theme ###

Name: Divi Child
Version: 1.0.0
Author URL: https://www.elegantthemes.com
Child Theme: ✔
Parent Theme Name: Divi
Parent Theme Version: 4.21.0
Parent Theme Author URL: http://www.elegantthemes.com
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Admin ###

Enabled Features: activity-panels
analytics
coupons
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
multichannel-marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: block-editor-feature-enabled
minified-js
new-product-management-experience
product-variation-management
settings

Daily Cron: ✔ Next scheduled: 2023-04-29 15:00:11 +02:00
Options: ✔
Notes: 76
Onboarding: completed

### Action Scheduler ###

Abgeschlossen: 354
Oldest: 2023-03-31 23:23:18 +0200
Newest: 2023-04-29 13:47:17 +0200

Fehlgeschlagen: 2
Oldest: 2023-04-17 17:07:46 +0200
Newest: 2023-04-29 13:43:30 +0200

Ausstehend: 3
Oldest: 2023-04-30 13:41:28 +0200
Newest: 2023-04-30 13:41:28 +0200

### Status report information ###

Generated at: 2023-04-29 13:49:27 +02:00
`

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

### PR DESCRIPTION
**Pull request by octaedro**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

This PR fixes the width of the button to set the variation prices in the legacy experience.

Before

Screenshot 2024-05-21 at 11 38 36 AM

Screenshot 2024-05-21 at 11 38 54 AM

After

Screenshot 2024-05-21 at 11 35 02 AM

Screenshot 2024-05-21 at 11 35 18 AM

Closes #38054.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Be sure you're using the legacy editor and you're in a site that uses the Deutsch `de_DE` language.
2. Create a variant product with an attribute (like size).
3. You will see an option that mentions the variations without price, which opens a modal.
4. Verify that the button looks correct.

Screenshot 2024-05-21 at 3 10 06 PM

5. Press the button and confirm that the `Add price` button inside the modal looks good. 

Screenshot 2024-05-21 at 3 10 17 PM

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce/client/legacy/css/admin.scss [hunk 1]
```diff
@@ -5137,10 +5137,6 @@ img.help_tip {
 
 			p.description {
 				margin-bottom: 8px;
-
-				&.is-error, span.is-error {
-					color: $red;
-				}
```
Comment (louwie17) [inline plugins/woocommerce/client/legacy/css/admin.scss]: I assume this is a rebase issue, this was actually added yesterday: https://github.com/woocommerce/woocommerce/commit/7c4ce7091604875d7d4278cb0f623f634ba597d3 and should be kept and not removed.

**Review by louwie17 — DISMISSED** [review]
The main fix itself tested well and that code looks good, just left one comment about some deleted code that should be reverted.

File: plugins/woocommerce/client/legacy/css/admin.scss (hunk 1 above)
Comment (octaedro) [reply plugins/woocommerce/client/legacy/css/admin.scss]: Yes, it was a rebase problem. I fixed it in the commit [08c7f9a](https://github.com/woocommerce/woocommerce/pull/47682/commits/08c7f9ad5bee6c1b32e0e15724d039bfc98c252f)

**Comment by octaedro** [comment]
Thank you, @louwie17, for the review! I’ve just fixed the issue caused by the rebase.
Could you take another look at this PR?

**Review by mdperez86 — APPROVED** [review]
LGTM @octaedro nice job!!!

**Pull request merged**